packages/common/src/components/TableView/sort.ts

```diff
@@ -76,7 +74,7 @@ export const useSort = (
     () =>
       compareWith(
         activeSort,
-        i18n.resolvedLanguage,
+        resolvedLanguage,
```

Comment:
this should be removed ... , sorting should return the same order for users using different languages, if we order by language, we may give different order of items for different languages